### ISSUE TITLE
Would be amazing if we could find a specific menu item near us with this api

### ISSUE THREAD

**Issue by elvinup** [asopinka/mcdonalds-api#1]
Still having trouble finding spicy mcchickens out here in the midwest :/

Would you consider adding a menu item search function in the future based on location? I'd find it helpful :)

**Comment by asopinka** [comment]
For sure! I'll look into it and see what I come up with and let you know.

**Comment by asopinka** [comment]
Bad news @elvinup :(

It doesn't look like McDonald's lists any of their specialty sandwiches on the site. For example, I know the north-east USA and East Canada have a McLobster for a few months in the summer. But there's no mention of which restaurants carry these specialty sandwiches, or when.

It seems they only list the items that are static on a menu across all locations in a country. I've visited Dubai, and the Spicy McChicken is an item always available, so it also appears on the [UAE McDonalds website](https://www.mcdonalds.com/ae/en-ae/product/spicy-mcchicken.html).

Might have to pack your bags 🛫 😏

**Comment by elvinup** [comment]
@asopinka 
Thanks for looking into it! I actually found very recently that the McDonald's app on android can find specific menu items for each location. I checked, and all the states surrounding my state have that spicy goodness, but not the 10 mcdonalds locations or so near me that I checked in my state 😢  lol. 

I guess it would still be cool to check the app's api requests to figure out which location closest has a specific item. I found an app, "Packet Capture", that lets me sniff requests on android when i use the mcdonalds app. I'm still trying to make sense of it, but the key to figuring it out could be in there.
